Geometry/HcalCommonData/interface/HcalDDDRecConstants.h

```diff
@@ -125,6 +126,8 @@ class HcalDDDRecConstants {
   unsigned int              layerGroupSize(int eta) const;
   unsigned int              layerGroup(int eta, int i) const;
 
+  static const int           maxLayer_=18;
```

Comment:
@bsunanda - it seems to me that Hcal DD constants were introduces for the exact purpose of avoiding hardcoded constants in the code